### ISSUE TITLE
Sync in the Background

### ISSUE THREAD

**Issue by laalsaas** [ankidroid/Anki-Android#14479]
**Is your feature request related to a problem? Please describe.**
I have automatic sync on opening the app configured. However, sometimes syncing takes up to a minute. Cancelling is not really better, since this will also take like 30secs. When I open the App to quickly capture a card, this blocks me from doing so.

**Describe the solution you'd like**
Syncing should happen in the background, without blocking the UI. There might be a notification with a progress bar or an animated sync Logo. However, UI should not be blocked by a sync happening.

**Describe alternatives you've considered**
I see three theoretical alternatives to my solutions.
1. Investigate why the sync takes so long when my collection isn't that huge. I have no idea about anki{,droid,sync} internals, so this might also be related to the protocol and not feasible to change
2. Turning off auto-sync. However, this would probably result in me forgetting to sync all the time.
3. Investigate *why* on earth *cancelling* a sync takes so long.

**Comment by BrayanDSO** [comment]
Duplicate of #12468

**Comment by BrayanDSO** [comment]
Hey there, I feel your issues as well. Syncing has changed in the 2.17 alphas and will have a different experience, although it has some issues too since 2.17 is still under development after all
